examples/waitForEvent.js

```diff
@@ -0,0 +1,41 @@
+import { browser } from 'k6/x/browser';
+
+export const options = {
+  scenarios: {
+    browser: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+            type: 'chromium',
+        },
+      },
+    },
+  },
+}
+
+export default async function() {
+  const context = browser.newContext()
+
+  // We want to wait for two page creations
+  // before carrying on.
+  var counter = 0
+  const promise = context.waitForEvent("page", { predicate: page => {
+    if (++counter == 2) {
+      return true
+    }
+    return false
+  } })
+  
+  // Now we create two pages.
+  const page = context.newPage()
+  const page2 = context.newPage()
+
+  // We await for the page creation
+  // events to be processed and the
+  // predicate to pass.
```

Comment:
Is there a reason to split these comment lines extremely short? I noticed a few of them in the Go code in this PR, too. I wouldn't mind it if this weren't a refactoring PR :) It's OK if it's your personal style :)

Reply:
Yeah, personal preference but it changes 😅 

I'll change them though so that they're consistent. I think around 80 characters max line length should be good enough as a guide.

Reply:
Resolved in 197378ca4f9662c5cdc6f49441f621e47b84cfec

Reply:
Perfect, thanks!